Vagrantfile

```diff
@@ -1,5 +1,6 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
+ENV['VAGRANT_DEFAULT_PROVIDER'] = 'virtualbox'
```

Comment:
Why forcing VirtualBox?

Reply:
We do have it also in the Readme as recommendation, so I just coded it to have a useful default. Otherwise the default varies to VirtualBox or VMwareFusion (on a Mac).